src/shogun/machine/IterativeMachine.h

```diff
@@ -69,9 +69,13 @@ namespace shogun
 			pb.complete();
 
 			if (m_complete)
+			{
 				SG_SINFO(
 				    "%s converged after %d iterations.\n", this->get_name(),
 				    m_current_iteration)
+
+				this->end_training();
```

Comment:
so this is called but NewtonSVM doesnt implement that or?
It could though if we recycle the variables mentioned above

Reply:
yeah newtonsvm does not implement it for now